utils/swift_build_support/swift_build_support/build_script_invocation.py

```diff
@@ -429,6 +429,11 @@ def convert_to_impl_arguments(self):
                 "--llvm-install-components=%s" % args.llvm_install_components
             ]
 
+        if args.build_lld:
+            impl_args += [
+                "--llvm-install-components=%s" % args.llvm_install_components
```

Comment:
Does llvm install components accumulate in build-script like tthis? Doesn't this break the usage of llvm-install-components above.

Reply:
Not sure how I managed to commit this, but this should have been 
```
if args.build_lld:
  impl_args += ["--build-lld"]
```
🤦